src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/HearingDocumentServiceTest.java

```diff
@@ -145,11 +146,12 @@ public void sendToBulkPrint() {
         verify(bulkPrintService).printApplicantDocuments(eq(caseDetails), eq(AUTH_TOKEN), bulkPrintDocumentsCaptor.capture());
         verify(bulkPrintService).printRespondentDocuments(eq(caseDetails), eq(AUTH_TOKEN), bulkPrintDocumentsCaptor.capture());
 
-        assertThat(bulkPrintDocumentsCaptor.getValue().size(), is(4));
+        assertThat(bulkPrintDocumentsCaptor.getValue().size(), is(5));
         assertThat(bulkPrintDocumentsCaptor.getValue().get(0).getBinaryFileUrl(), is(BINARY_URL));
         assertThat(bulkPrintDocumentsCaptor.getValue().get(1).getBinaryFileUrl(), is(BINARY_URL));
         assertThat(bulkPrintDocumentsCaptor.getValue().get(2).getBinaryFileUrl(), is(BINARY_URL));
         assertThat(bulkPrintDocumentsCaptor.getValue().get(3).getBinaryFileUrl(), is(BINARY_URL));
+        assertThat(bulkPrintDocumentsCaptor.getValue().get(4).getBinaryFileUrl(), is(BINARY_URL));
```

Comment:
Could these be in a loop based on bulkPrintDocumentsCaptor.getValue().size() to save having to add a new line each time a new one is added?

Reply:
Change made